hw/bsp/pinetime/syscfg.yml

```diff
@@ -0,0 +1,53 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Settings this BSP defines.
+syscfg.defs:
+
+# Settings this BSP overrides.
+syscfg.vals:
```

Comment:
Any reasons for not having `CONFIG_FCB_FLASH_AREA`, `REBOOT_LOG_FLASH_AREA` and `COREDUMP_FLASH_AREA` set ?
Also there are no timer configurations for BLE controller, is there a reason for that?

Reply:
No, I don't have a reason for that, other than that is was not in the [BSP porting tutorial](https://mynewt.apache.org/latest/os/core_os/porting/port_bsp.html). Do you have a good example of that?

Reply:
I created a PR for the blinky tutorial: https://github.com/apache/mynewt-documentation/pull/91

And I added the flash areas as with most other BSP.

Reply:
Yes, take look at [bsp/nordic_pca10040](https://github.com/apache/mynewt-core/blob/master/hw/bsp/nordic_pca10040) . You will probably need to upbate your `syscfg.yml` and `hal_bsp_get_nvic_priority` to be able to build and run [bleprph](https://github.com/apache/mynewt-nimble/tree/master/apps/bleprph).